packages/Switch/src/Switch.styles.js

```diff
@@ -8,10 +8,10 @@ const KnobPositionStyles = {
     transform: translateX(12px);
   `,
   medium: css`
-    transform: translate(20px, 0);
+    transform: translate(16px, 0);
```

Comment:
Normally we use `$space` token, or `stylers.spacer` helper... but these seem pretty arbitrary so i think how you did it is fine.

Reply:
I could change that. Should i replace all of the width and height values with spacers as opposed to px as well?

Reply:
I dont think so, as some would be like 2 spacer + 1/2 spacer